data/gwc/geowebcache-diskquota.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<gwcQuotaConfiguration>
+  <enabled>true</enabled>
+  <cacheCleanUpFrequency>10</cacheCleanUpFrequency>
+  <cacheCleanUpUnits>SECONDS</cacheCleanUpUnits>
+  <maxConcurrentCleanUps>2</maxConcurrentCleanUps>
+  <globalExpirationPolicyName>LFU</globalExpirationPolicyName>
+  <globalQuota>
+    <value>500</value>
+    <units>MiB</units>
+  </globalQuota>
+  <quotaStore>H2</quotaStore>
```

Comment:
So this is basically using a 500MB in memory h2 database for caching the type of files defined in the other xml file in this PR? I'm 👍 regardless but I'm also somewhat curious to check back in on this in production to confirm whether it makes a difference. I'm not familiar with geoserver to have any evidence based feelings about this.

Reply:
Nah, the disk quota is stored on disk. I have specifically disabled the in memory cache for now. The difference a cache makes for tiling is pretty significant. The usage patterns tend to result in quite a few cache hits over the short term.

Reply:
H2 is an in memory database though...

Reply:
I guess maybe it doesn't have to be, but that's all I've ever seen it used for. Regardless, this is all good lol